pkg/retry/retry.go

```diff
@@ -9,6 +9,10 @@ const (
 	Interval = 15 * time.Second
 	// Timeout is the total time we will wait for an event to occur.
 	Timeout = time.Minute * 10
+	// SSHConnTimeout is the time we will wait for a handshake to occur when trying to establish an SSH connection
```

Comment:
nit:  There may be others elements involved here, not only the `handshake`. For instances, name resolution, system timeouts, etc.
Consider:
`// SSHConnTimeout is the maximum amount of time for the SSH connection to establish.`

Reply:
Thanks, reworded